dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -27,13 +28,30 @@ let cachedOphan: typeof window.guardian.ophan;
 /**
  * Loads Ophan (if it hasn't already been loaded) and returns a promise of Ophan's methods.
  */
-export const getOphan = async (): Promise<
-	NonNullable<typeof window.guardian.ophan>
-> => {
+export const getOphan = async (
+	renderingTarget: RenderingTarget,
+): Promise<NonNullable<typeof window.guardian.ophan>> => {
 	if (cachedOphan) {
 		return cachedOphan;
 	}
 
+	if (renderingTarget === 'Apps') {
+		cachedOphan = {
+			setEventEmitter: () => undefined, // We don't currently have a custom eventEmitter on DCR - like 'mediator' in Frontend.
+			trackComponentAttention: () => undefined,
+			record: (event) =>
+				log(
+					'dotcom',
+					"🧿 We're not recording this event because we are in DCAR",
+					event,
+				),
```

Comment:
Should this be a call to Sentry? Do we want to record this incorrect invocations somewhere?

Reply:
Good shout 👍

Reply:
Not entirely sure if Sentry is defined in the “Apps” rendering target…